justfile

```diff
@@ -37,7 +37,7 @@ flutter-deps:
     flutter pub get
 
 # Fetches rust dependencies
-rust-deps:
+rust-deps: install-async-bindgen
```

Comment:
It's more  a dart dependency, tbh.

Like it's used only by dart not by rust, it just happen to be written in rust.

But one option worth considering when (if) improving the story around async bindgen is to port the CLI tool to dart, making it a `build_runner` based tool.

Reply:
I didn't put for dart because sometimes we install only dart stuff on some job without rust. Otherwise we have that dart = true also imply rust = true.
 
Let's see about rewriting it because if we move the message handler directly in rust we might not need it.